scripts/get_maintainer.py

```diff
@@ -88,9 +89,39 @@ def parse_maintainers():
     return subsystems
 
 
+# If @patchset is a patchset files and contains 2 patches or more, write
+# individual patches to temporary files and return the paths.
+# Otherwise return [].
+def split_patchset(patchset):
+    psname = os.path.basename(patchset).replace('.', '_')
+    patchnum = 0
+    of = None
+    ret = []
+    try:
+        with open(patchset, "r") as f:
+            for line in f:
+                match = re.search(PATCH_START, line)
+                if match:
+                    # New patch found: create new file
+                    patchnum += 1
+                    prefix = "{}_{}_".format(patchnum, psname)
+                    of = tempfile.NamedTemporaryFile(mode="w", prefix=prefix,
+                                                     suffix=".patch",
+                                                     delete=False)
+                    ret += [of.name]
+                of.write(line)
+    except Exception:
+        pass
+    if len(ret) >= 2:
+        return ret
+    if ret and ret[0]:
```

Comment:
There are several ways to skin a cat, but since we already checked for "2 and above", I'd do:
```suggestion
    if len(ret) == 1:
```

Reply:
Yeah much better